src/components/suggestion/SuggestionSearch.tsx

```diff
@@ -0,0 +1,103 @@
+import { ListSuggestSearchFromVinted } from '../../api/api';
+import React, { useEffect, useState } from 'react';
+import CardSuggestionSearch from './CardSuggestionSearch';
+import { ChevronRight, ChevronLeft } from 'lucide-react';
+
+const SuggestionSearch: React.FC = () => {
+  const [scrollCards, setScrollCard] = useState<'left' | 'right' | 'both'>('right');
```

Comment:
Message de SonarLint : useState call is not destructured into value + setter pairsonarlint(typescript:S6754)

C'est parce qu'il manque un s à `setScrollCard`ou il y a un s en trop dans `scrollCards`. Les nom de la variable et de son setter doivent être pareils

Reply:
Corrigé